go/pkg/client/tree_test.go

```diff
@@ -1573,3 +1574,40 @@ func TestComputeOutputsToUploadDirectories(t *testing.T) {
 		})
 	}
 }
+
+func randomBytes(n int) []byte {
```

Comment:
nit: I think it is better to use function local rand state in benchmark.

Reply:
I'm not sure what you mean?

Reply:
I meant to use https://golang.org/pkg/math/rand/#Rand

Reply:
Done